<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recherche</class>
 <widget class="QDialog" name="recherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rechercher / Remplacer</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Principal">
    <item>
     <widget class="QFrame" name="frame_rechercher">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>351</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="layout_rechercher">
        <item>
         <widget class="QLineEdit" name="texte_rechercher"/>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_rechercher">
          <property name="text">
           <string>Rechercher</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_remplacer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>351</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="layout_remplacer">
        <item>
         <widget class="QLineEdit" name="texte_remplacer"/>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_remplacer">
          <property name="text">
           <string>Remplacer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_annuler">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>351</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="layout_annuler">
        <item>
         <widget class="QPushButton" name="bouton_annuler">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
